Makefile

```diff
@@ -1,6 +1,6 @@
 VENV_NAME?=venv
 PIP?=pip
-PYTHON?=python3
+PYTHON?=python3.10
```

Comment:
What's the thinking here? When we want to use a version other than 3.10 this means we need to `make pyright PYTHON=<some/other/python>` (or whatever the syntax is for providing variables to Make, I forget) instead of being able to use pyenv

Reply:
We do https://github.com/stripe/stripe-python/pull/1134/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR104

Reply:
I was thinking about the local dev environment and just pointing out that this will supersede pyenv.

Reply:
It won't supersede pyenv AFAIK, it will always create pyenv with python3.10 instead of whatever the current version of python in environment is

Reply:
But yeah, it's harder to create pyenv using a different version now

Reply:
I think you are hearing `venv` when I am saying `pyenv`?

I guess part of my workflow when I want to locally run pyright or pytest on a particular python version is
```
$ echo "3.x" > .python-version
$ make 
$ . venv/bin/activate
$ pyright
$ python -m pytest tests/
```
and now `.python-version` will be ignored, unless I override the explicit hardcoding of 3.10
```
$ echo "3.x" > .python-version
$ make PYTHON=python
$ . venv/bin/activate
$ pyright
$ python -m pytest tests/
```

Reply:
>I think you are hearing venv when I am saying pyenv?
I do :/  Sorry.

How often do we use the ability to switch python versions?